### PR TITLE
Add MFA

### PR DESCRIPTION
**Pull request by tenmaster**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by volbap** [comment]
+1 @mpaktiti

File: articles/native-platforms/android/09-mfa.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description: This tutorial will show you how to configure Multi Factor Authentication (MFA) via Google Authenticator in your app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
```
Comment (mpaktiti) [inline articles/native-platforms/android/09-mfa.md:18]: @tenmaster Missing the download seed project panel

File: articles/native-platforms/android/09-mfa.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### 1. Enable MFA in your account
```
Comment (mpaktiti) [inline articles/native-platforms/android/09-mfa.md:19]: @tenmaster Before this paragraph we need an intro text explaining what MFA is. You can copy and paste most of it from another quickstart, for example [angular2](https://auth0.com/docs/quickstart/spa/react/09-mfa)

File: articles/native-platforms/android/09-mfa.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}) %>
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+  
+<%= include('../_includes/_mfa-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/android/09-mfa.md:17]: @tenmaster There is no `_includes` folder at `https://github.com/auth0/docs/tree/master/articles/native-platforms`

**Pull request closed without merging**